hnn_core/network.py

```diff
@@ -1128,10 +1235,18 @@ class _Connectivity(dict):
         Cell type of source gids.
     target_type : str
         Cell type of target gids.
+    gid_pairs : dict
+        dict indexed by src gids with the format:
+        {src_gid: [target_gids, ...], ...}
+        where each src_gid indexes a list of all its targets.
     num_srcs : int
         Number of unique source gids.
     num_targets : int
         Number of unique target gids.
+    src_range : range
+        Range of gids identified by src_type.
+    target_range : target_range
```

Comment:
```suggestion
    target_range : range
```

the datatype should be after the column